fflas-ffpack/fflas/fflas_sparse/read_sparse.h

```diff
@@ -447,8 +447,8 @@ namespace FFLAS {
         file.write((char*) &magic     , sizeof(int64_t)) ;
         file.write((char*) &field_desc, sizeof(int64_t)) ;
         file.write((char*) &data_type , sizeof(int64_t)) ;
-        file.write((char*) &rowdim , sizeof(int64_t)) ;
-        file.write((char*) &coldim , sizeof(int64_t)) ;
+        file.write((const char*) &rowdim , sizeof(int64_t)) ;
```

Comment:
These edits are unrelated with this PR. Can you remove them, and possibly open another one ? Thanks